dotCMS/src/main/java/com/dotcms/rest/api/v2/contenttype/FieldResource.java

```diff
@@ -157,13 +154,10 @@ public Response getContentTypeFieldById(@PathParam("typeId") final String typeId
             @PathParam("fieldId") final String fieldId, @Context final HttpServletRequest req)
             throws DotDataException, DotSecurityException {
 
```

Comment:
why was this call `webResource.init` deleted?

Reply:
was marked as unused

Reply:
restored